reference-implementation/__tests__/resolving.js

```diff
@@ -63,6 +60,53 @@ describe('Unmapped', () => {
   });
 });
 
+describe('the empty string', () => {
+  it('should fail for an unmapped empty string', () => {
+    const resolveUnderTest = makeResolveUnderTest(`{}`);
+    console.warn = () => {};
```

Comment:
Why is this necessary? Same below. It seems we should not get a warning if we're also getting a TypeError... But even if we do, we should test for the presence of the warning.

Reply:
The warning comes from parsing and the thrown `TypeError` comes from resolution. They're composed in `makeResolveUnderTest`. I'll assert that the warnings are present in the same way we do in `testWarningHandler`.